modules/core/keeper/msg_server.go

```diff
@@ -217,7 +217,7 @@ func (k Keeper) ChannelOpenTry(goCtx context.Context, msg *channeltypes.MsgChann
 	}
 
 	// Perform 04-channel verification
-	channelID, cap, err := k.ChannelKeeper.ChanOpenTry(ctx, msg.Channel.Ordering, msg.Channel.ConnectionHops, msg.PortId, msg.PreviousChannelId,
```

Comment:
Should we add a comment for [this field in the proto definition of `MsgChannelOpenTry`](https://github.com/cosmos/ibc-go/blob/main/proto/ibc/core/channel/v1/tx.proto#L84) that this field has been deprecated and will be ignored by core IBC?